services.go

```diff
@@ -63,6 +63,7 @@ type InvoicesService interface {
 	GetPDF(invoiceNumber int, language string) (*Response, *bytes.Buffer, error)
 	Preview(accountCode string) (*Response, *Invoice, error)
 	Create(accountCode string, invoice Invoice) (*Response, *Invoice, error)
+	Collect(invoiceNumber int) (*Response, *Invoice, error)
```

Comment:
Added in [e7ca00](https://github.com/blacklightcms/recurly/pull/21/commits/fcbd89a3b82ff5a45a6e012e560a106c60e7ca00).